src/pytest/test_flow.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2018-2020, Michael P. Howard
+# Copyright (c) 2021-2024, Auburn University
+# Part of azplugins, released under the BSD 3-Clause License.
+
+import collections
+
+import hoomd
+import numpy
+import pytest
+from hoomd.conftest import pickling_check
+import hoomd.azplugins
+
+
+def test_constant_flow_field(simulation_factory):
+    # make the force
+    U = hoomd.azplugins.flow.ConstantFlow(mean_velocity=(1, 0, 0))
+    numpy.testing.assert_array_almost_equal(U.mean_velocity, (1, 0, 0))
+    pickling_check(U)
+
+    U.mean_velocity = (1, 1, 1)
```

Comment:
I would use different values, in case you accidentally copy pasted an index wrong.
```suggestion
    U.mean_velocity = (1, 2, 3)
```